app/models/service.rb

```diff
@@ -137,6 +137,14 @@ def power_status
     options[:power_status]
   end
 
+  def self.active
+    Service.where(:lifecycle_state => ["provisioning", "provisioned"], :retired => false)
+  end
+
+  def self.inactive
+    Service.where(:lifecycle_state => ["unprovisioned", "error_in_provisioning"], :retired => false) + Service.retired
```

Comment:
Database-wise I think you can do the following which will turn it into a single query:

```suggestion
    Service.where(:lifecycle_state => ["unprovisioned", "error_in_provisioning"], :retired => false).or(Service.retired)
```